projects/project-player-library/src/lib/pages/details-page/details-page.component.html

```diff
@@ -2,6 +2,7 @@
   <div class="flex justify-between">
     <span class="details-card-header">
         <h4 class="break-all">{{projectDetails?.title || "Undefined"}}</h4>
+        <p class="pl-2 break-all text-base text-gray-500">{{projectDetails?.programName}}</p>
```

Comment:
Is it the same styling which is used in the preview-page?

Reply:
yes
padding i have added

Reply:
is that required?

Reply:
removed it